ugs-core/src/com/willwinder/universalgcodesender/AbstractController.java

```diff
@@ -684,15 +690,18 @@ private synchronized void flushQueuedCommands() {
 
     // Reset send queue and idx's.
     private void flushSendQueues() {
-        this.errorCount = 0;
-        this.numCommands -= this.getRowStat(RowStat.ROWS_REMAINING);
+        errorCount = 0;
```

Comment:
The error count is never used so I was tempted to remove its usage. But if it's implemented it could be pretty useful.